### PR TITLE
[Task#47120] CRUD post

### PR DESCRIPTION
**Pull request by methanol-dev**
## Related Tickets:
- [#47120](https://edu-redmine.sun-asterisk.vn/issues/47120)

## WHAT this PR does?
- CRUD post

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by methanol-dev** [comment]
ready

File: app/Http/Controllers/AdminPostController.php [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+        Storage::disk('public')->put('post/' . $imageName, $img);
+
+        $post = new Post();
+        $post->title = $request->title;
+        $post->user_id = Auth::id();
+        $post->slug = $slug;
+        $post->images = $imageName;
+        $post->content = $request->content;
+        $post->status = Post::APPROVED;
+        $post->save();
```
Comment (ghost) [inline app/Http/Controllers/AdminPostController.php]: dùng hàm create ở đây đc ko?

**Comment by methanol-dev** [comment]
ready

**Comment by methanol-dev** [comment]
ready

File: app/Http/Controllers/AdminPostController.php [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+        Storage::disk('public')->put('post/' . $imageName, $img);
+
+        Post::create([
+            'title' => $request->title,
+            'user_id' => Auth::id(),
+            'slug' => $slug,
+            'images' => $imageName,
+            'content' => $request->content,
+            'status' => Post::APPROVED
```
Comment (ghost) [inline app/Http/Controllers/AdminPostController.php]:
```suggestion
            'status' => Post::APPROVED,
```

**Comment by methanol-dev** [comment]
ready

**Pull request merged**